src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionIntroViewModel.swift

```diff
@@ -30,20 +30,13 @@ class ExposureSubmissionIntroViewModel {
 		return DynamicTableViewModel.with {
 			$0.add(.section(
 				header: .image(
-					UIImage(named: "Illu_Submission_Funktion1"),
+					UIImage(named: "Illu_Test_registration"),
 					accessibilityLabel: AppStrings.ExposureSubmissionDispatch.accImageDescription,
 					accessibilityIdentifier: AccessibilityIdentifiers.General.image,
 					height: 200
 				),
 				separators: .none,
-				cells: [
-					.body(
-						text: AppStrings.ExposureSubmissionDispatch.description,
```

Comment:
Same here: Are the String and AccessibilityIdentifier still used by someone else? If not, we could remove them...

Reply:
The AccessibilityIdentifiers are not used anywhere else. Didn't find any UITest using this.